doc/build/dts/intro-syntax-structure.rst

```diff
@@ -0,0 +1,472 @@
+      ``length`` is the number of bytes occupied by the registers.
+    - I2C devices (like ``apds9960@39`` and its siblings):
+      ``address`` is a slave address on the I2C bus. There is no ``length``
+      value.
+    - SPI devices: ``address`` is a chip select line number; there is no
+      ``length``.
+
+    You may notice some similarities between the ``reg`` property and common
+    unit addresses described above. This is not a coincidence. The ``reg``
+    property can be seen as a more detailed view of the addressable resources
+    within a device than its unit address.
+
+status
+    A string which describes whether the node is enabled.
+
+    The devicetree specification allows this property to have values
+    ``"okay"``, ``"disabled"``, ``"reserved"``, ``"fail"``, and ``"fail-sss"``.
+    Only the values ``"okay"`` and ``"disabled"`` are currently relevant to
+    Zephyr; use of other values currently results in undefined behavior.
+
+    A node is considered enabled if its status property is either ``"okay"`` or
+    not defined (i.e. does not exist in the devicetree source). Nodes with
+    status ``"disabled"`` are explicitly disabled. (For backwards
+    compatibility, the value ``"ok"`` is treated the same as ``"okay"``, but
+    this usage is deprecated.) Devicetree nodes which correspond to physical
+    devices must be enabled for the corresponding ``struct device`` in the
+    Zephyr driver model to be allocated and initialized.
+
+interrupts
+    Information about interrupts generated by the device, encoded as an array
+    of one or more *interrupt specifiers*. Each interrupt specifier has some
+    number of cells. See section 2.4, *Interrupts and Interrupt Mapping*, in the
+    `Devicetree Specification release v0.3`_ for more details.
+
+    Zephyr's devicetree bindings language lets you give a name to each cell in
+    an interrupt specifier.
+
+.. _Devicetree Specification release v0.3:
+   https://www.devicetree.org/specifications/
+
+.. highlight:: none
+
+.. note::
+
+   Earlier versions of Zephyr made frequent use of the ``label`` property,
+   which is distinct from the standard :ref:`node label <dt-node-labels>`.
+   Use of the label *property* in new devicetree bindings, as well as use of
+   the :c:macro:`DT_LABEL` macro in new code, are actively discouraged. Label
+   properties continue to persist for historical reasons in some existing
+   bindings and overlays, but should not be used in new bindings or device
+   implementations.
+
+.. _dt-writing-property-values:
+
+Writing property values
+***********************
+
+This section describes how to write property values in DTS format. The property
+types in the table below are described in detail in :ref:`dt-bindings`.
+
+Some specifics are skipped in the interest of keeping things simple; if you're
+curious about details, see the devicetree specification.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 4 4
+
+   * - Property type
```

Comment:
Duplicate of what is on the bindings-syntax page (which has additional rows)

Reply:
Yep, that's on purpose. `path` and `compound` aren't worth covering in the introduction in my opinion.